lesson22/test/playwright1.test.spec.js

```diff
@@ -0,0 +1,72 @@
+const { chromium } = require("playwright");
+const chai = require('chai')
+const expect = chai.expect
+const { MainPage } = require('../pageobject/MainPage');
+const { BaseElements } = require('../helpers/baseElements');
+const { Navbar } = require('../pageobject/pagecomponents/Navbar')
+const { Footer } = require('../pageobject/pagecomponents/Footer');
+
+
+
+describe("Automation tests with playwright", () => {
+  let browser, page, main, context, base, navbar, footer;
+
+  before(async () => {
+    browser = await chromium.launch({ headless: false, args: ['--start-maximized'] });
+    context = await browser.newContext();
+    page = await context.newPage('https://playwright.dev');
+    main = new MainPage(page);
+    base = new BaseElements(page);
+    navbar = new Navbar(page);
+    footer = new Footer(page);
+  });
+
+  after(async () => {
+    await page.close();
+    await browser.close();
+  });
+
+ 
+
+  it("Checks the title of the page", async () => {
```

Comment:
title of the page should be Fast and reliable end-to-end testing for modern web apps | Playwright